### PR TITLE
Feedback fix

### PR DESCRIPTION
**Pull request by vinay-ThoughtWorks**

### CODE REVIEW

File: ui/react-components/components/Service/ServiceTypeSearch.jsx [hunk 1]
```diff
@@ -25,15 +27,19 @@ const ServiceTypeSearch = props => {
         }
     }, [serviceUuid]);
 
+
     const getServiceType = async (serviceUuid) => {
         const service = await getService(serviceUuid);
         const serviceTypes = service.serviceTypes;
-        setServiceTypes(createDropdownOptions(serviceTypes));
+        if(serviceTypes.length === 0) {
+            setIsServiceNotSelected(true);
```
Comment (shivarach) [inline ui/react-components/components/Service/ServiceTypeSearch.jsx]: `serviceTypes` would be enough and `isServiceNotSelected` can be removed?

File: ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx [hunk 2]
```diff
@@ -25,6 +25,16 @@ const AppointmentEditorCommonFieldsWrapper = props => {
     const componentsDisableStatus = props.componentsDisableStatus || {};
     const errorTranslations = getErrorTranslations(intl);
 
+    const updateLocationBasedOnService = (selectedService) => {
+        isEmpty(selectedService.value.location) ? updateAppointmentDetails({location: null})
```
Comment (shivarach) [inline ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: Possibility of `undefined`?

Comment (AlekhyaYalla) [inline ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: Should we add a null check for selectedService?

**Comment by shivarach** [comment]
I've done the review for 814 and 815, added few comments

File: ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx [hunk 3]
```diff
@@ -14,15 +14,18 @@ import {getAppointmentConflictDetails} from "./ConflictsUtil";
...
-        return (
-            <div className={classNames(conflictMessage)}>Current {currentAppointmentService} request
-                <span className={classNames(boldContent)}> conflicts with {existingAppointmentService} </span>
-                appointment on
-            </div>
-        );
+      let formattedMessage = <FormattedMessage id="PATIENT_DOUBLE_BOOKING_CONFLICT_MESSAGE"
+                                               defaultMessage={doubleBookingConflictMessage}
+                                               values={{
+                                                 currentAppointmentService,
```
Comment (AlekhyaYalla) [inline ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx]: Do we need '{{' double parenthesis?

File: ui/react-components/components/SuccessModal/SuccessModal.jsx [hunk 4]
```diff
@@ -29,11 +29,13 @@ const SuccessModal = (props) => {
             </div>
             <div>
                 <h1 className={classNames(saveModalTitle)}>
-                    <FormattedMessage id={'APPOINTMENT_SAVE_SUCCESS_TITLE'} defaultMessage={isEdit ? 'Update Successful!' : 'Save successful'}/>
+                    <FormattedMessage id={'APPOINTMENT_SAVE_SUCCESS_TITLE'} defaultMessage={'Save successful'}/>
```
Comment (AlekhyaYalla) [inline ui/react-components/components/SuccessModal/SuccessModal.jsx]: We showing the same message for both add & edit?

Comment (vinay-ThoughtWorks) [reply ui/react-components/components/SuccessModal/SuccessModal.jsx]: We have seperate modal in edit. So removed here.

File: ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx (hunk 3 above)
Comment (vinay-ThoughtWorks) [reply ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx]: yes.  We are passing object here.

File: ui/react-components/components/EditAppointment/EditAppointment.jsx [hunk 5]
```diff
@@ -238,7 +238,8 @@ const EditAppointment = props => {
     const checkAndUpdateRecurringAppointments = async (applyForAllInd) => {
         if (isValidRecurringAppointment()) {
             const recurringRequest = getRecurringAppointmentRequest(applyForAllInd);
-            const response = await getRecurringAppointmentsConflicts(recurringRequest);
+            const response = applyForAllInd ? await getRecurringAppointmentsConflicts(recurringRequest) :
```
Comment (binduak) [inline ui/react-components/components/EditAppointment/EditAppointment.jsx]: @AlekhyaYalla what does "applyForAllInd" mean ?? I feel variable name is not meaningful..

File: ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx (hunk 2 above)
Comment (binduak) [reply ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: I see this is already been handled in the below line
https://github.com/Bahmni/openmrs-module-appointments-frontend/blob/feedback-fix/ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx#L66

File: ui/react-components/components/Service/ServiceTypeSearch.jsx (hunk 1 above)
Comment (binduak) [reply ui/react-components/components/Service/ServiceTypeSearch.jsx]: Good one.. I will try that once

Comment (binduak) [reply ui/react-components/components/Service/ServiceTypeSearch.jsx]: I have tried it but we are setting value for isServiceNotSelected in state and its not UseEffect.  so the intention of disabling serviceType is not met

File: ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx (hunk 3 above)
Comment (AlekhyaYalla) [reply ui/react-components/components/Conflicts/PatientDoubleBookingConflicts.jsx]: ok

File: ui/react-components/components/SuccessModal/SuccessModal.jsx (hunk 4 above)
Comment (AlekhyaYalla) [reply ui/react-components/components/SuccessModal/SuccessModal.jsx]: ok

File: ui/react-components/components/EditAppointment/EditAppointment.jsx (hunk 5 above)
Comment (AlekhyaYalla) [reply ui/react-components/components/EditAppointment/EditAppointment.jsx]: applyForAllIndicator it is. existing variable,

File: ui/react-components/components/AppointmentNotes/AppointmentNotes.jsx [hunk 6]
```diff
@@ -1,12 +1,12 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import {notes} from './AppointmentNotes.module.scss'
 
 const AppointmentNotes = (props) => (
-        <div>
+        <Fragment>
             <textarea value={props.value || ''} className={classNames(notes)} onChange={props.onChange} data-testid="notes"/>
```
[the remark this answers is not in the document]
Comment (shivarach) [reply ui/react-components/components/AppointmentNotes/AppointmentNotes.jsx]: @vinay-ThoughtWorks @AlekhyaYalla

File: ui/react-components/components/EditAppointment/EditAppointment.jsx (hunk 5 above)
Comment (binduak) [reply ui/react-components/components/EditAppointment/EditAppointment.jsx]: hmm..difficult to get the context by name. I think we should handle the name change in the later part of the cards. I am noting it down.

File: ui/react-components/components/AppointmentNotes/AppointmentNotes.jsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (binduak) [reply ui/react-components/components/AppointmentNotes/AppointmentNotes.jsx]: @shivarach MaxLength change included with later commits, that is why you don't get to see here.

**Pull request merged**